notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/JPANotificationRESTController.java

```diff
@@ -0,0 +1,253 @@
+package org.jasig.portlet.notice.controller.rest;
+
+import org.jasig.portlet.notice.rest.AddresseeDTO;
+import org.jasig.portlet.notice.rest.EntryDTO;
+import org.jasig.portlet.notice.rest.EventDTO;
+import org.jasig.portlet.notice.service.jpa.IJpaNotificationRESTService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import java.util.Set;
+
+
+/**
+ * Controller for the REST endpoints exposed to manage
+ * JPA notifications.
+ *
+ * @author Josh Helmer, jhelmer.unicon.net
+ * @since 3.0
+ */
+@Controller
+@RequestMapping(JPANotificationRESTController.REQUEST_ROOT)
+public class JPANotificationRESTController {
+    private Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
Given that public class can be sub-classed, suggest `protected` logger so available to potential sub-classes.